### PR TITLE
Fixed IP lookup issue for the Client End-Point objects

### PR DESCRIPTION
**Pull request by sajagana**
Note:

Added support to read Client End-Point objects based on the fvIp address value.

### CODE REVIEW

File: aci/data_source_aci_fvcep.go [hunk 1]
```diff
@@ -253,11 +278,27 @@ func dataSourceAciClientEndPointRead(d *schema.ResourceData, m interface{}) erro
...
+
+		macPattern := regexp.MustCompile("cep-(.+)/")
+		tempQueryString := ""
+		for i := 0; i < len(fvIpDns); i++ {
+			if tempQueryString == "" {
+				tempQueryString = fmt.Sprintf("eq(fvCEp.mac, \"%s\")", macPattern.FindStringSubmatch(fvIpDns[i])[1])
+			} else {
+				tempQueryString = fmt.Sprintf("%s,eq(fvCEp.mac, \"%s\")", tempQueryString, macPattern.FindStringSubmatch(fvIpDns[i])[1])
+			}
+		}
```
Comment (lhercot) [inline aci/data_source_aci_fvcep.go:294]: Wouldn't this be nicer if we did:
```
                tempQueryString := fmt.Sprintf("eq(fvCEp.mac, \"%s\")", macPattern.FindStringSubmatch(fvIpDns[0])[1])
		for i := 1; i < len(fvIpDns); i++ {
			tempQueryString = fmt.Sprintf("%s,eq(fvCEp.mac, \"%s\")", tempQueryString, macPattern.FindStringSubmatch(fvIpDns[i])[1])
		}
```

**Review by lhercot — CHANGES_REQUESTED** [review]

File: website/docs/d/client_end_point.html.markdown [hunk 2]
```diff
@@ -11,39 +11,49 @@ description: |-
 
 Data source for ACI Client End Point
 
+## API Information ##
+
+* `Class` - fvCEp
+* `Distinguished Name` - uni/tn-{tenant_name}/ctx-{ctx_name}/cep-{client_end_point_name}
+* `Distinguished Name` - uni/tn-{tenant_name}/ap-{ap_name}/epg-{epg_name}/cep-{client_end_point_name}
+* `Distinguished Name` - uni/tn-{tenant_name}/l2out-{l2out_name}/instP-{instance_profile_name}/cep-{client_end_point_name}
```
Comment (anvitha-jain) [inline website/docs/d/client_end_point.html.markdown:19]: Does this terraform module accept all the 3 Dns?

**Review by anvitha-jain — CHANGES_REQUESTED** [review]

File: website/docs/d/client_end_point.html.markdown (hunk 2 above)
Comment (sajagana) [reply website/docs/d/client_end_point.html.markdown:19]: As per the code, this module should accept all 3 Dns. But I couldn’t test it.

**Review by lhercot — APPROVED** [review]
LGTM

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Pull request merged**